src/content/configuration/optimization.mdx

```diff
@@ -56,7 +56,6 @@ module.exports = {
     minimizer: [
       new TerserPlugin({
         parallel: true,
-        sourceMap: true, // Must be set to true if using source-maps in production
```

Comment:
I think it still takes that option, but as part of the `terserOptions`-object:

```
new TerserPlugin({
  parallel: true,
  terserOptions: {
    sourceMap: true, // Must be set to true if using source-maps in production
  },
})
```

Reply:
When I have `devtool: 'source-map'` set in webpack's config and I build with production mode, it will provide sourcepmap input for minify and inside Terser Plugins folder /lib/minify.js after line 240 it will set the terserOptions for you:
```
  if (inputSourceMap) {
    // @ts-ignore
    terserOptions.sourceMap = {
      asObject: true
    };
  }
```
So I think you are right that you can set it by yourself, but still that it is unnecessary to set that option manually and then IMO it is better to leave it out of Webpack documentation?

Reply:
ps. I'm unable to re-request your @jaylinski review. This is an one of my first PRs for open source projects so please guide me if I'm not following the right procedures :smile:

Reply:
> but still that it is unnecessary to set that option manually and then IMO it is better to leave it out of Webpack documentation?

Agree :) (I only commented on this because I found the same issue with the documentation. I'm not an official reviewer, this was just a comment.)

Reply:
@jaylinski I don't think there's any official ones :) Anyone can help reviewing.

@tokkoro The CLA bot is broken at the moment. I've asked the author for help, stay tuned.

Reply:
CLA site was working and I accepted the license. Now I'm wating to it to show up here :)